framework/content/ecl-tables/ecl-tables.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Tables related behaviors.
+ */
+
+// eslint-disable-next-line import/prefer-default-export
+export function eclTables() {
```

Comment:
What if you want to apply this transformation not on all tables but only on a subset? My two cents: you could add an optional parameter ` eclTables(elements = null)` and then reuse it `const tables = elements === null ? document.getElementsByClassName('ecl-table') : elements;`. Or you could pass the the query selector and then call `querySelectorAll` instead of `getElementsByClassName` :)